presto-cli/src/main/java/com/facebook/presto/cli/Console.java

```diff
@@ -133,7 +133,7 @@ public void run()
                 Optional.ofNullable(clientOptions.truststorePath),
                 Optional.ofNullable(clientOptions.truststorePassword),
                 Optional.ofNullable(clientOptions.user),
-                Optional.ofNullable(promptFor(clientOptions.password)),
+                getPassword(),
```

Comment:
Didn't you forget about prompt here?
Hidden `-P` but what about `--password` which existed before?